api/src/main/resources/emails/unused_disk/content.html

```diff
@@ -41,7 +41,7 @@
   user <span class="variable">${DISK_CREATOR_USERNAME}</span>. These charges are billed
   according to the workspace's billing account, which is currently
   <span class="variable">${BILLING_ACCOUNT_DETAILS}</span>. This disk is currently
-  <span class="variable">${DISK_STATUS}</span> the <span class="variable">${APP_NAME} application.</span>
+  <span class="variable">${DISK_STATUS}</span> the <span class="variable">${ENVIRONMENT_TYPE} application.</span>
```

Comment:
it's not the app *name* - that's more like a UUID.  This should also be "environment" rather than app because Jupyter is possible.